### PR TITLE
chore: updates exhibition guide and team info

### PR DESCRIPTION
**Pull request by farhaanbukhsh**

### CODE REVIEW

File: src/_data/team.json [hunk 1]
```diff
@@ -270,12 +270,12 @@
     "instagram": "instagram.com/kybrdbnd"
   },
   {
-    "name": "Farhaan",
-    "image": "/image/ZadraSol.webp",
+    "name": "Farhaan Bukhsh",
+    "image": "https://avatars.githubusercontent.com/u/7670449?s=400&u=bf6eb65bdf25c98310e3711ca1d8b58a84ae47d6&v=4",
```
Comment (anistark) [inline src/_data/team.json:274]: Add image to `src/image/team` and give that link.

File: src/_data/team.json [hunk 2]
```diff
@@ -411,11 +411,11 @@
   },
   {
     "name": "Shabnam",
-    "image": "/image/ZadraSol.webp",
+    "image": "https://x.com/iishabbyii/photo",
```
Comment (anistark) [inline src/_data/team.json]: also image update in local.

**Review by anistark — CHANGES_REQUESTED** [review]

File: src/_data/team.json (hunk 1 above)
Comment (farhaanbukhsh) [reply src/_data/team.json:274]: @anistark  above images are being loaded from GitHub only :P

File: src/_data/team.json (hunk 2 above)
Comment (farhaanbukhsh) [reply src/_data/team.json]: I have changed it to githubs profile image

File: src/_data/team.json (hunk 1 above)
Comment (anistark) [reply src/_data/team.json:274]: it'll be on server. site isn't hosted on github

**Review by anistark — CHANGES_REQUESTED** [review]

File: src/_data/team.json (hunk 1 above)
Comment (farhaanbukhsh) [reply src/_data/team.json:274]: @anistark check this https://github.com/pythonindia/inpycon2024/pull/126/files#diff-81a59159f57c56e3665a2579f2c014836025bf6172c7065d15a40ef2181eb193R92-R101

Comment (SaptakS) [reply src/_data/team.json:274]: Github links will work because those are public URLs.

Comment (anistark) [reply src/_data/team.json:274]: sure. I thought we were downloading each image. Hence suggested.

Just realised it wasn't for team. My bad. Too much around. Got confused.

**Review by anistark — APPROVED** [review]

**Pull request merged**